apl-conf/src/main/resources/conf-tn1/apl-blockchain.properties

```diff
@@ -763,4 +763,6 @@ apl.allowUpdates=true
 # Total number of public keys in a genesisAccounts.json file
 apl.genesisAccounts.publicKeyNumberTotal=230730
 # Total number of balances in a genesisAccounts.json file
-apl.genesisAccounts.balanceNumberTotal=84832
\ No newline at end of file
+apl.genesisAccounts.balanceNumberTotal=84832
+
+apl.noshardimport=true
```

Comment:
This lacks following description.
Value affects node's behavior after start up with empty database. 'true' means nodes will NOT try to download latest 'shard archive' from network and will load all data starting from genesis account. 'false' means making an attempt to load 'latest shard archive' from network and start from it on higher height, but it will fall back to loading genesis data in case unsuccessful attempt.

Reply:
Let it be for a while